src/org/thoughtcrime/securesms/NicknameHandler.java

```
@@ -42,7 +42,14 @@ public NicknameHandler setRecipientDatabase(RecipientDatabase recipientDatabase)
      */
     public boolean setNickname(Recipient recipient, String nickname) {
         this.setupDatabaseHandler();
-        return this.recipientDatabase.setNickname(recipient, nickname);
+        try{ return this.recipientDatabase.setNickname(recipient, nickname);}
+        catch (Exception e) { System.err.println("Did not add nickname"); return false;}
```

It's probably not a great Idea to silently ignore exceptions